packages/documentation/docs/controls/_forms_custom_field_styleguide.md

```diff
@@ -0,0 +1,43 @@
+The custom field is a wrapper component that can host any other forms component. It provides a set of properties that allow you to control the validation state of the field, as well as the helper text that is displayed below the field. The custom field is a versatile tool to create your own form fields, that can be used in combination with the 'Form' components to create complex forms.
+
+![Custom field](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3303-3291&t=DlxXBQ9vTnyDcIUI-4 )
+
+1. Label
+2. Helper text/Feedback text
+3. Form components
+4. Required indicator
+%%
+## Quick guide
+
+![Custom field_new](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3482-7078&t=DlxXBQ9vTnyDcIUI-4)
+
+- **Usage:** Use the custom field when your desired solution is not covered by the standard form field component.
+- **Customize as Needed:** Add specific form components (e.g., input fields, buttons) to meet your use case.
+- **Helper and feedback text:** The custom field provides the option to display helper and feedback text automatically.
+- **Follow Guidelines:** Refer to the form field and validation chapters for detailed guidelines on labels, helper text, and feedback text.
+%%
+## Options
+
+- **Label:** Refer to the form field chapter for detailed guidelines.
+- **Helper text**: Refer to the form field chapter for detailed guidelines.
+- **Feedback text**: Refer to the form field chapter for detailed guidelines.
+- **Customization**: Add form components to create the use case you need. For example, for a file upload field, add an input field with a `readonly` state and an [icon button](buttons/icon-button.md).
+
+![Custom field example](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3483-7223&t=DlxXBQ9vTnyDcIUI-4)
+
+## Behavior in context
+
+- **Validation:** Refer to the validation chapter for detailed guidelines.
```

Comment:
(1) add link, (2) we had the reference sentence different in the other forms chapters, please make them consistent